gen_import_utils.py

```diff
@@ -80,21 +80,49 @@ def separate_titles(row, config):
     return row
 
 
+def validate_date(date_string):
```

Comment:
There are several existing libraries for this; punch in your code and ask the new chatgpt model about them. Dates are subtle and quick to anger; it's always best to leverage existing libraries when you can.